packages/eslint-config-airbnb/rules/react.js

```diff
@@ -8,12 +8,22 @@ module.exports = {
   'rules': {
     // Prevent missing displayName in a React component definition
     'react/display-name': 0,
+    // Forbid certain propTypes (any, array, object)
+    'react/forbid-prop-types': 0,
     // Enforce boolean attributes notation in JSX
     'react/jsx-boolean-value': 2,
+    // Validate closing bracket location in JSX
+    'react/jsx-closing-bracket-location': 2,
```

Comment:
Thanks - please use a configuration of `{ "nonEmpty": "tag-aligned", "selfClosing": "tag-aligned" }` such that in a multiline component declaration, `/>` should always be on its own line, aligned with the opening tag.